Healthy/Classes/Modules/Splash/OnboardingViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  OnboardingViewController.swift
+//  Healthy
+//
+//  Created by dodo on 15/05/2023.
+//
+
+import UIKit
+
+class OnboardingViewController : UIViewController {
+    
+    let stackView = UIStackView()
+    
+    let imageView = UIImageView()
+    let titleLabel = UILabel()
+    let subtitleLabel = UILabel()
+    
+    init(imageName: String, titleText: String, subtitleText: String) {
+        super.init(nibName: nil, bundle: nil)
+        imageView.image = UIImage(named: imageName)
+
+        titleLabel.text = titleText
+        subtitleLabel.text = subtitleText
+    }
```

Comment:
we can make a struct to hold all this information and inject it (just for now because it also the mission of viewModel)
`The view shouldn't know anything about the data`
```
struct OnBoardingModel {
       var imageName: String
       var titleText: String
       var subtitleText: String
}
```